drivers/auth/auth_mod.c

```diff
@@ -267,38 +262,32 @@ static int auth_nvctr(const auth_method_param_nv_ctr_t *param,
 	/* Parse the DER encoded integer */
 	assert(data_ptr);
 	p = (char *)data_ptr;
-	if (*p != ASN1_INTEGER) {
-		/* Invalid ASN.1 integer */
+	if (*p != ASN1_INTEGER)		/* Invalid ASN.1 integer */
 		return 1;
-	}
 	p++;
 
 	/* NV-counters are unsigned integers up to 32-bit */
 	len = (unsigned int)(*p & 0x7f);
-	if ((*p & 0x80) || (len > 4)) {
+	if (*p & 0x80 || len > 4)
```

Comment:
@masahir0y Ahhh! Really sorry missed this one in my previous review; apologies! Could you revert the parentheses on this one too?
Thanks!